crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -1178,6 +1178,23 @@ impl Backend {
         if let Some(factory) = &self.precompile_factory {
             inject_precompiles(&mut evm, factory.precompiles());
         }
+        let cheats_arc = Arc::new(self.cheats.clone());
```

Comment:
we can clone cheatsmanager because this is already just and arc wrapper so we don't need to wrap this